utils/make-phar.php

```diff
@@ -16,55 +16,41 @@
 /**
  * Adds a file to the PHAR
  *
- * @param [Phar]   $phar Phar archive resource
- * @param [string] $path Path to the file to add
- * @return [void]
+ * @param Phar   $phar Phar archive resource
+ * @param string $path Path to the file to add
+ * @param array $options Options to alter the file being added
+ *      integer offset The number of lines to skip at the beginning of the file
  */
-function addFile($phar, $path)
+function addFile($phar, $path, $options = ['offset' => 0,])
 {
     $key = str_replace('./', '', $path);
-
     if (!BE_QUIET) {
         echo "$key - $path\n";
     }
-
-    $phar[ $key ] = file_get_contents($path);
+    $file_contents = file_get_contents($path);
+    if (isset($options['offset']) && (boolean)$options['offset']) {
+        $file_contents = implode("\n", array_slice(explode("\n", $file_contents), $options['offset']));
+    }
+    $phar[$key] = $file_contents;
 }
 
 $phar = new Phar(DEST_PATH, 0, 'terminus.phar');
 
 $phar->startBuffering();
 
-// PHP files
 $finder = new Finder();
-$finder
-    ->files()
+$finder->files()
     ->ignoreVCS(true)
     ->in('./assets')
-    ->in('./php')
-    ->in('./vendor')
     ->in('./config')
-    ->exclude('test')
-    ->exclude('tests')
-    ->exclude('Tests')
-    ->exclude('php-cli-tools/examples');
-
-foreach ($finder as $file) {
-    addFile($phar, $file);
-}
-
-// Non-PHP Files
-$finder = new Finder();
-$finder
-    ->files()
-    ->ignoreVCS(true)
-    ->ignoreDotFiles(false)
-    ->in('./templates');
+    ->in('./src')
+    ->in('./vendor');
 
 foreach ($finder as $file) {
     addFile($phar, $file);
 }
 
+addFile($phar, './bin/terminus', ['offset' => 1,]);
```

Comment:
Need to hack off the top line of /bin/terminus to prevent it from echoing when running.